x-pack/plugins/monitoring/public/components/table/eui_table.js

```diff
@@ -22,6 +22,10 @@ export class EuiMonitoringTable extends React.PureComponent {
       search.box['data-test-subj'] = 'monitoringTableToolBar';
     }
 
+    if (search.box && !search.box.schema) {
```

Comment:
I didn't find any issues related to this but I'm curious about what it's accomplishing. It doesn't conform to the proptypes specified in the docs, so for my own purposes I'm wondering what it does.

Reply:
Ah, so I'm just allowing any future monitoring tables to specify a schema manually, but if they didn't, set the `schema` property to true so the EUI code does it automatically.